Makefile

```
@@ -41,79 +45,39 @@ fix: ## Fixup files in the repo.
-	./hack/image/push-manifest-list.sh $(HELM_IMAGE) ${HELM_ARCHES}
-
-.PHONY: image-build-scorecard-test image-push-scorecard-test image-push-scorecard-test-multiarch
-SCORECARD_TEST_CI_IMAGE = quay.io/operator-framework/scorecard-test
-SCORECARD_TEST_IMAGE ?= $(SCORECARD_TEST_CI_IMAGE)
-SCORECARD_TEST_ARCHES:="amd64" "ppc64le" "arm64" "s390x"
-image-build-scorecard-test:
-	./hack/image/build-scorecard-test-image.sh $(SCORECARD_TEST_CI_IMAGE):dev
-image-push-scorecard-test:
-	./hack/image/push-image-tags.sh $(SCORECARD_TEST_CI_IMAGE):dev $(SCORECARD_TEST_IMAGE)-$(shell go env GOARCH)
-image-push-scorecard-test-multiarch:
-	./hack/image/push-manifest-list.sh $(SCORECARD_TEST_IMAGE) ${SCORECARD_TEST_ARCHES}
-
-.PHONY: image-build-scorecard-test-kuttl image-push-scorecard-test-kuttl image-push-scorecard-test-kuttl-multiarch
-SCORECARD_TEST_KUTTL_CI_IMAGE = quay.io/operator-framework/scorecard-test-kuttl
-SCORECARD_TEST_KUTTL_IMAGE ?= $(SCORECARD_TEST_KUTTL_CI_IMAGE)
-SCORECARD_TEST_KUTTL_ARCHES:="amd64" "ppc64le" "arm64"
-image-build-scorecard-test-kuttl:
-	./hack/image/build-scorecard-test-kuttl-image.sh $(SCORECARD_TEST_KUTTL_CI_IMAGE):dev
-image-push-scorecard-test-kuttl:
-	./hack/image/push-image-tags.sh $(SCORECARD_TEST_KUTTL_CI_IMAGE):dev $(SCORECARD_TEST_KUTTL_IMAGE)-$(shell go env GOARCH)
-image-push-scorecard-test-kuttl-multiarch:
-	./hack/image/push-manifest-list.sh $(SCORECARD_TEST_KUTTL_IMAGE) ${SCORECARD_TEST_KUTTL_ARCHES}
-
-.PHONY: image-build-custom-scorecard-tests
-CUSTOM_SCORECARD_TESTS_CI_IMAGE = quay.io/operator-framework/custom-scorecard-tests
-CUSTOM_SCORECARD_TESTS_IMAGE ?= $(CUSTOM_SCORECARD_TESTS_CI_IMAGE)
-image-build-custom-scorecard-tests:
-	./hack/image/build-custom-scorecard-tests-image.sh $(CUSTOM_SCORECARD_TESTS_CI_IMAGE):dev
+.PHONY: build
+build: ## Build operator-sdk, ansible-operator, and helm-operator.
+	@mkdir -p $(BUILD_DIR)
+	go build $(GO_BUILD_ARGS) -o $(BUILD_DIR) ./cmd/{operator-sdk,ansible-operator,helm-operator}
+
+# Build scorecard binaries.
+.PHONY: build/scorecard-test build/scorecard-test-kuttl build/custom-scorecard-tests
+build/scorecard-test build/scorecard-test-kuttl build/custom-scorecard-tests:
+	go build $(GO_GCFLAGS) $(GO_ASMFLAGS) -o $(BUILD_DIR)/$(@F) ./images/$(@F)
+.PHONY: build/operator-sdk build/ansible-operator build/helm-operator
+build/operator-sdk build/ansible-operator build/helm-operator:
+	go build $(GO_BUILD_ARGS) -o $(BUILD_DIR)/$(@F) ./cmd/$(@F)
+
+##@ Dev images
+
+# Convenience wrappers for building/pushing all images.
+.PHONY: image-build
+IMAGE_TARGET_LIST := operator-sdk helm-operator ansible-operator scorecard-test scorecard-test-kuttl custom-scorecard-tests
+image-build: $(foreach i,$(IMAGE_TARGET_LIST),image/$(i)) ## Build all images.
+
+# Build an image.
+IMAGE_REPO ?= quay.io/operator-framework
+image/%: build/%
+	rm -rf build/_image && mkdir -p build/_image && cp build/$* build/_image
+	docker build -t $(IMAGE_REPO)/$*:dev -f ./images/$*/Dockerfile build/_image
```

Should we cleanup after ourselves after the image build as well?
```suggestion
	docker build -t $(IMAGE_REPO)/$*:dev -f ./images/$*/Dockerfile build/_image
	rm -rf build/_image
```